test/PredictiveDeployer.t.sol

```diff
@@ -90,14 +94,42 @@ contract PredictiveDeployerTest is Test {
         assertEq(actualChild, expectedChild);
     }
 
-    function test_CannotDeployReplay(uint256 pkNum) public {
+    // helper function
+    function deployChild(VmSafe.Wallet memory wallet, bytes memory bytecode, bytes32 hashedBytecode)
```

Comment:
I like the idea of a helper function, but let's move helper functions into a HelperContract?

Something like:

```
contract HelperContract {
    function helperFunction1() public {}
    function helperFunction1() public {}
}

contract PredictiveDeployerTest is HelperContract {}
```

The helper contract can be in a separate directory within the test directory. Something like `test/helpers/HelperContract.sol`

We can change the name, `HelperContract`, to whatever makes sense. `DeploymentHelper`? Open to ideas.